marketplace/deployer_util/overlay_test_schema.py

```diff
@@ -41,6 +44,12 @@ def main():
   dest = load_yaml(args.dest)
   if 'properties' in dest:
     for prop in orig['properties']:
+      if (deep_get(orig, 'properties', prop, 'x-google-marketplace', 'type') !=
+          deep_get(dest, 'properties', prop, 'x-google-marketplace', 'type')):
+        log(
+            "{} WARNING Changing x-google-marketplace type is not allowed. Property: {}",
```

Comment:
shouldn't this be an fatal error instead of warning?

Reply:
I was originally inclined to treat as a warning and let the test itself make the decision. But maybe it is better to fail fast here.